cms/settings/base.py

```diff
@@ -628,7 +626,7 @@
 # The Django default for the maximum number of GET or POST parameters is 1000. For
 # especially large Wagtail pages with many fields, we need to override this. See
 # https://docs.djangoproject.com/en/3.2/ref/settings/#data-upload-max-number-fields
-DATA_UPLOAD_MAX_NUMBER_FIELDS = int(env.get("DATA_UPLOAD_MAX_NUMBER_FIELDS", 1000))
+DATA_UPLOAD_MAX_NUMBER_FIELDS = int(env.get("DATA_UPLOAD_MAX_NUMBER_FIELDS", 10_000))
```

Comment:
Any specific reason for this change?.

Reply:
edit forms for pages with loats of StreamField blocks can hit the 1k limit. I am pre-empting the "related datasets" addition with it.

6.4 made 10k the default in for the start template